kuma/core/tests/test_helpers.py

```diff
@@ -50,7 +50,7 @@ def test_message_with_url_is_link(self):
         m = Message(message='Go to http://bit.ly/sample-demo', is_global=True,
                     is_active=True, url='/')
         m.save()
-        ok_('Go to <a href="http://bit.ly/sample-demo">'
+        ok_('Go to <a href="http://bit.ly/sample-demo" rel="noopener">'
```

Comment:
According to `[jinja2 policy](http://jinja.pocoo.org/docs/2.10/api/#policies)`, seems like `noopener` is default